modules/installation-user-defined-tags-azure.adoc

```diff
@@ -0,0 +1,109 @@
+// * installing/installing_azure/installing-azure-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-azure-user-defined-tags_{context}"]
+= User-defined tags for Azure
+
+In {prodcut-ttitle} you can use the Azure user-defined tags for grouping resources and for managing resource access and cost. You can define tags on the Azure resources in the `install-config.yaml` file only during {product-title} cluster creation. You cannot modify the user-defined Azure tags after cluster creation.
+
+Support for user-defined Azure tags is available only in {product-title} 4.13 as a Technology Preview (TP) and for the resources created on Azure Public Cloud. User-defined Azure tags are not supported for {product-title} clusters upgraded to {product-title} 4.13.
+
+You can apply user-defined and {product-title} specific tags only to the resources created by the {product-title} installer and its core operators such as Machine api provider azure Operator, Cluster Ingress Operator, Cluster Image Registry Operator.
+
+You can use the `.platform.azure.userTags` field in the `install-config.yaml` file to define the list of user-defined Azure tags. You must set the `featureSet` field as `TechPreviewNoUpgrade` in the `install-config.yaml` file.
+
+The user-defined Azure tags have the following limitations:
+
+* A tag key can have a maximum of 128 characters.
+* A tag key must begin with a letter, end with a letter, number or underscore, and can contain only letters, numbers, underscores, periods, or hyphens.
+* Tag keys are case-insensitive.
+* Tag keys cannot be `name`. It cannot have prefixes such as `kubernetes.io`, `openshift.io`, `microsoft`, `azure`, `windows`.
+* A tag value can have a maximum of 256 characters.
+* You can configure a maximum of 10 tags for resource group and resources.
+
+.Sample `install-config.yaml` file
+[source,yaml]
+----
+additionalTrustBundlePolicy: Proxyonly <1>
+apiVersion: v1
+baseDomain: catchall.azure.devcluster.openshift.com <2>
+featureSet: TechPreviewNoUpgrade <3>
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane:  <4>
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  platform: {}
+  replicas: 3
+metadata:
+  creationTimestamp: null
+  name: bhb <5>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  azure:
+    baseDomainResourceGroupName: os4-common <6>
+    cloudName: AzurePublicCloud <7>
+    outboundType: Loadbalancer
+    region: southindia <8>
+    userTags:
+      createdBy: bhb <9>
+      environment: dev <10>
+----
+<1>
+<2>
+<3>
+<4>
+<5>
+<6>
+<7>
+<8>
+<9>
+<10>
+
+After creating the cluster, You can refer to the `.status.platformStatus.azure.resourceTags` in the `config.openshift.io/Infrastructure` resource `cluster` object to know the list of the defined tags.
```

Comment:
In general, should we provide a command for this or above would be enough?

Reply:
I think we can provide a command- that will clarify the things

Reply:
Below command can be used to query resourceTags in Infrastructure object.
`oc get infrastructures.config.openshift.io cluster -o=jsonpath-as-json='{.status.platformStatus.azure.resourceTags}'`

Reply:
Addressed